js/languagePickerView.js

```diff
@@ -22,7 +22,8 @@ define([
             var data = this.model.toJSON();
             var template = Handlebars.templates[this.constructor.template];
             this.$el.html(template(data));
```

Comment:
yea, it's a pain that it works like this, but in some ways it's Backbones best solution to a messy problem.